annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java

```diff
@@ -141,6 +148,56 @@ public AnnotationController(VariantAnnotationService variantAnnotationService,
        return getVariantAnnotation(variants, isoformOverrideSource);
     }
 
+    @ApiOperation(value = "getHotspotAnnotation",
```

Comment:
@zheins: Good point. We may want to send a single query and get everything we need at once.

On the other hand, I think the ultimate goal is to avoid using genome-nexus in pipelines at all, but instead query data on the fly.